src/components/studies/StudyBuilder.tsx

```diff
@@ -162,14 +157,17 @@ const StudyBuilder: FunctionComponent<StudyBuilderProps> = ({
         setHasObjectChanged(false)
       },
     )*/
+
     try {
       const newVersion = await StudyService.updateStudy(study, token!)
       const updatedStudy = { ...study, version: newVersion }
       setData({
         ...builderInfo,
         study: updatedStudy,
       })
-
+      if (error !== '') {
```

Comment:
do we need this check?

Reply:
Yes, if we do not do this then the error message will still show on top of the screen even if the study was updated successfully.

Reply:
What I'm questioning is the needs to check.  I think just doing setError('') is sufficient. Yes it's extra set if it's already empty but it removes the check an decreases mental load for a person who might need to look at the code later. It's really a tiny detail.